session.go

```diff
@@ -4,43 +4,40 @@ import (
 	"reflect"
 	"sync"
 
-	"upper.io/db"
+	"upper.io/db.v2"
+	"upper.io/db.v2/postgresql" // TODO: Figure out how to remove this and make it generic.
```

Comment:
I'll give it some thought too. First I'm going to try to get the tests passing. 

One idea, (maybe will work) .. is to connect with upper directly, then pass the db.Database to bond.UseDatabase() for example, and remove the .Open() method..